.travis.yml

```diff
@@ -2,6 +2,10 @@
 ---
 dist: xenial
 language: python
+env:
+  global:
+    - LSR_ANSIBLES='ansible==2.6.* ansible==2.7.* ansible==2.8.*'
```

Comment:
Why is this configured in travis.yml and not in config.sh? In case somone wants to run molecule outside of travis, this needs to be defined as well, doesn't it? This also affects the variable on the next line.

Reply:
`config.sh` should contain only parts that may vary across repositories. Running molecule outside Travis can be performed via `LSR_ANSIBLE_DEP=<ansible-dep> LSR_MSCENARIO=<molecule scenario> tox -e molecule` (just `tox -e molecule` runs molecule with the newest Ansible Python package installed and with the default scenario). Furthermore, running molecule for all supported Ansible versions and for all supported scenarios outside of Travis by one command will be imho possible only by introducing a shell script that wraps `tox` command (this possibility is worth of discussion). Maybe it should be good to write a documentation for our Travis and `tox` based CI and put it to file named like `TOX.INI.README.md`.